src/renderer/modals/Receive/steps/StepReceiveFunds.js

```diff
@@ -155,23 +156,30 @@ const StepReceiveFunds = ({
 
   const confirmAddress = useCallback(async () => {
     try {
-      if (!device) {
-        throw new DisconnectedDevice();
-      }
-      const { address } = await command("getAddress")({
-        derivationMode: mainAccount.derivationMode,
-        currencyId: mainAccount.currency.id,
-        devicePath: device.path,
-        path: mainAccount.freshAddressPath,
-        verify: true,
-      }).toPromise();
-      if (address !== mainAccount.freshAddress) {
-        throw new WrongDeviceForAccount(`WrongDeviceForAccount ${mainAccount.name}`, {
-          accountName: mainAccount.name,
-        });
+      if (getEnv("MOCK")) {
+        setTimeout(() => {
+          onChangeAddressVerified(true);
+          transitionTo("receive");
+        }, 3000);
```

Comment:
(for later) we should have the opportunity to mock that part. it bypass the potential test of having WrongDeviceForAccount and also a delay during the device verification

maybe a future system to mock commands? anyway, not now

Reply:
we can still emit `{type:"error",error:{name:"WrongDeviceForAccount"}}` and trigger it 🤷